NetBSD CI build is broken

```
FAILED: librz/debug/librz_debug.so.0.8.0.p/p_native_bsd_bsd_debug.c.o 
clang -Ilibrz/debug/librz_debug.so.0.8.0.p -I. -I.. -Ilibrz -I../librz -Ilibrz/include -I../librz/include -I../librz/bin/format/elf -I../librz/bin/format/dmp -I../librz/bin/format/mdmp -I../librz/bin/format/pe -I../subprojects/rzgdb/include -I../subprojects/rzgdb/include/gdbclient -I../subprojects/rzgdb/include/gdbserver -Isubprojects/rzwinkd -I../subprojects/rzwinkd -Ilibrz/util/sdb/src -I../librz/util/sdb/src -I../librz/bin/format -I../librz/arch/isa -I../librz/arch/isa_gnu -Ilibrz/arch -I../librz/arch -I../librz/type/parser -I../subprojects/rzqnx/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O2 -g -DRZ_PLUGIN_INCORE=1 -DSUPPORTS_PCRE2_JIT -D_GNU_SOURCE --std=gnu99 -Werror=sizeof-pointer-memaccess -fvisibility=hidden -fPIC -MD -MQ librz/debug/librz_debug.so.0.8.0.p/p_native_bsd_bsd_debug.c.o -MF librz/debug/librz_debug.so.0.8.0.p/p_native_bsd_bsd_debug.c.o.d -o librz/debug/librz_debug.so.0.8.0.p/p_native_bsd_bsd_debug.c.o -c ../librz/debug/p/native/bsd/bsd_debug.c
../librz/debug/p/native/bsd/bsd_debug.c:68:24: error: variable has incomplete type 'struct ptrace_lwpinfo'
        struct ptrace_lwpinfo linfo = { 0 };
                              ^
../librz/debug/p/native/bsd/bsd_debug.c:68:9: note: forward declaration of 'struct ptrace_lwpinfo'
        struct ptrace_lwpinfo linfo = { 0 };
               ^
../librz/debug/p/native/bsd/bsd_debug.c:70:19: error: use of undeclared identifier 'PT_LWPINFO'; did you mean 'P_LWPID'?
        int ret = ptrace(PT_LWPINFO, dbg->pid, (char *)&linfo, sizeof(linfo));
                         ^~~~~~~~~~
                         P_LWPID
/usr/include/sys/idtype.h:45:2: note: 'P_LWPID' declared here
        P_LWPID,        /* An LWP identifier. */
        ^
../librz/debug/p/native/bsd/bsd_debug.c:81:24: error: use of undeclared identifier 'PL_EVENT_NONE'
        if (linfo.pl_event == PL_EVENT_NONE) {
                              ^
3 errors generated.
```
See full log at https://builds.sr.ht/~thestr4ng3r/job/1236382#task-build

--- a/librz/debug/p/native/bsd/bsd_debug.c
+++ b/librz/debug/p/native/bsd/bsd_debug.c
@@ -64,9 +64,10 @@ static void addr_to_string(struct sockaddr_storage *ss, char *buffer, int buflen
 
 int bsd_handle_signals(RzDebug *dbg) {
 #if __KFBSD__ || __NetBSD__
+	siginfo_t siginfo;
+#if __KFBSD__
 	// Trying to figure out a bit by the signal
 	struct ptrace_lwpinfo linfo = { 0 };
-	siginfo_t siginfo;
 	int ret = ptrace(PT_LWPINFO, dbg->pid, (char *)&linfo, sizeof(linfo));
 	if (ret == -1) {
 		if (errno == ESRCH) {
@@ -83,7 +84,6 @@ int bsd_handle_signals(RzDebug *dbg) {
 		return 0;
 	}
 
-#if __KFBSD__
 	siginfo = linfo.pl_siginfo;
 #else
 	struct ptrace_siginfo sinfo = { 0 };
@@ -108,6 +108,13 @@ int bsd_handle_signals(RzDebug *dbg) {
 	case SIGSEGV:
 		dbg->reason.type = RZ_DEBUG_REASON_SEGFAULT;
 		break;
+#if __NetBSD__
+	case SIGTRAP:
+		if (siginfo.si_code == TRAP_BKPT) {
+			dbg->reason.type = RZ_DEBUG_REASON_BREAKPOINT;
+		}
+		break;
+#endif
 	}
 
 	return 0;

--- a/librz/debug/p/native/bsd/bsd_debug.c
+++ b/librz/debug/p/native/bsd/bsd_debug.c
@@ -110,7 +110,7 @@ int bsd_handle_signals(RzDebug *dbg) {
 		break;
 #if __NetBSD__
 	case SIGTRAP:
-		if (siginfo.si_code == TRAP_BKPT) {
+		if (siginfo.si_code == TRAP_BRKPT) {
 			dbg->reason.type = RZ_DEBUG_REASON_BREAKPOINT;
 		}
 		break;

